routes/status.js

```diff
@@ -46,13 +53,30 @@ async function _checkDynamoConnection () {
       msg: null
     }
   } catch (err) {
+    console.log(err)
     return {
       status: false,
       msg: `Failed to connect to DynamoDB with ${err.code}`
     }
   }
 }
 
+async function _checkEcConnection () {
+  try {
+    const data = await getDynamo()
+
+    return {
+      url: data.ECurl || '',
+      status: data.ECurl ? true : false  // TODO: figure out how to see if this is connected or not
```

Comment:
If you can pass the `req.session` down to here, you might try logging it out and seeing if it has any information on the engine being used. Then this value would essentially be checking that the app is currently using redis or not. I think that makes sense.